demisto_sdk/commands/common/tools.py

```diff
@@ -564,14 +564,25 @@ def find_type(path: str):
 
 def get_common_server_path(env_dir):
     common_server_dir = get_common_server_dir(env_dir)
-
     return os.path.join(common_server_dir, 'CommonServerPython.py')
 
 
-def get_common_server_dir(env_dir):
-    common_server_pack_path = os.path.join(env_dir, 'Packs', 'Base', 'Scripts', 'CommonServerPython')
-    common_server_script_path = os.path.join(env_dir, 'Scripts', 'CommonServerPython')
+def get_common_server_path_pwsh(env_dir):
+    common_server_dir = get_common_server_dir_pwsh(env_dir)
+    return os.path.join(common_server_dir, 'CommonServerPowerShell.py')
```

Comment:
```suggestion
    return os.path.join(common_server_dir, 'CommonServerPowerShell.ps1')
```
I think this should be the correct one.